FosServer/src/main/java/com/example/fostracker/servlets/FosTrackerServlet/GetStoresVerifiedByAgentServlet.java

```diff
@@ -0,0 +1,125 @@
+
+        // Reads the json String using POST request.
+        String json = agentVerifiedStoreDetailsRequest.getReader().readLine();
+
+        // sets the response to json format.
+        response.setContentType("application/json");
+        response.setCharacterEncoding("UTF-8");
+
+        // output variable is used to write to response.
+        PrintWriter output;
+        output = response.getWriter();
+
+        if (json == null) {
+            response.setStatus(403);
+            output.println("No data given");
+        } else {
+
+            // Convert json string to json object to parse data.
+            JsonObject jsonObj = new Gson().fromJson(json, JsonObject.class);
+
+            // Get email from the JSON object.
+            String agentEmail = jsonObj.get("agentEmail").getAsString();
+
+            // storeAndCoordinateDataIterator is a Store object that is used to store the row we are iterating.
+            Store storeAndStatusDataIterator;
+            // storeAndCoordinateDataIteratorString stores the storeAndCoordinateDataIterator object as json String.
+            String storeAndStatusDataIteratorString;
+
+
+            // Querying the database table and storing in storeAndCoordinateData.
+            ResultSet storeAndStatusData = VerificationDatabaseHelper.queryStoreAndStatusDataByEmail(agentEmail);
+
+            // If storeAndStatusData is not empty then prints all the rows else prints "No data exists".
+            if (storeAndStatusData.next()) {
+                // stores the indexes of columns in storeAndCoordinateData ( ResultSet object ).
+                int columnStorePhoneIndex =
+                        storeAndStatusData.getColumnIndex(VerificationDatabaseHelper.COLUMN_STORE_PHONE);
+                int columnStoreStatusIndex =
+                        storeAndStatusData.getColumnIndex(VerificationDatabaseHelper.COLUMN_VERIFICATION_STATUS);
+                // Loop through all rows and stores the row data in storeAndCoordinateDataIterator. Converts storeAndCoordinateDataIterator into
+                // json object and prints to the screen.
+                do {
+
+                    storeAndStatusDataIterator = new Store(storeAndStatusData.getString(columnStorePhoneIndex),
+                            StoreDatabaseHelper.queryStoreCoordinatesUsingStorePhone(
+                                    storeAndStatusData.getString(columnStorePhoneIndex)),
+                            getStatusInt(storeAndStatusData.getString(columnStoreStatusIndex)
+                    ));
+                    storeAndStatusDataIteratorString = this.gson.toJson(storeAndStatusDataIterator);
+                    output.println(storeAndStatusDataIteratorString);
+                } while (storeAndStatusData.next());
+                storeAndStatusData.close();
+//                response.setStatus(200);
```

Comment:
Might want to consider defining global constants for these which you all can use.

Reply:
okay